### PR TITLE
Update ar.m3u

### LINKED ISSUE #7699: Add: LTO Televisión
### Channel Name

LTO Televisión

### Channel ID

_No response_

### Website

https://lastejasonline.blogspot.com/p/lto-television.html

### Stream URL

https://ltocatamarca.github.io/ltotv.m3u8

### Notes

Our channel transmits from Catamarca, Argentina

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by sguinetti**
This PR add streams of channels from Argentina.

Fixes #7699

### CODE REVIEW

File: streams/ar.m3u [hunk 1]
```diff
@@ -154,21 +154,161 @@ https://5f700d5b2c46f.streamlock.net/vertv/vertv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="CPEtv.ar" status="online",CPEtv
+https://stream.arcast.live/cpe/ngrp:cpe_all/playlist.m3u8
+#EXTINF:-1 tvg-id="ComarcaTV.ar" status="online",Comarca TV
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/canalccordoba
+#EXTINF:-1 tvg-id="CL3Cablevision.ar" status="online",CL3 Cablevision
+https://videostream.shockmedia.com.ar:1936/cl3cable/cl3cable/playlist.m3u8
+#EXTINF:-1 tvg-id="ChilecitoTV.ar" status="online",Chilecito TV
+https://vivo.solumedia.com:19360/grupoemail/grupoemail.m3u8
+#EXTINF:-1 tvg-id="CatamarcaTV.ar" status="error",Catamarca TV (288p)
+https://5f700d5b2c46f.streamlock.net/catamarcatelevision/catamarcatelevision/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/ar.m3u:302]: Supposing Catamarca TV is not 24/7, since both links are down.

**Review by BellezaEmporium — DISMISSED** [review]

File: streams/ar.m3u (hunk 1 above)
Comment (sguinetti) [reply streams/ar.m3u:302]: I guess. But the old link is possibly outdated, would be necessary check it again in the future.

**Comment by sguinetti** [comment]
Updated.

**Review by BellezaEmporium — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Orbe is most certainly one of the first Argentinian channels that has 50 fps on video and a good bandwidth.

**Review by freearhey — APPROVED** [review]

**Pull request merged**